### PR TITLE
Feature/delete target

### PR DESCRIPTION
**Pull request by agustinkoll-rootstrap**
### What's in

The possibility to delete a target with the new bottoms 
The target is removed from the map when the request is successful 

### How it looks

Screenshot_1612552735

Screenshot_1612795939

### CODE REVIEW

File: app/src/main/java/com/rootstrap/android/ui/activity/main/targetpoint/CreateTargetView.kt [hunk 1]
```diff
@@ -124,35 +150,100 @@ class CreateTargetView(
...
+        with(bindingRoot) {
+            selectedTarget = targetModel
+            showBottomButtons(false)
+            title_edit_text.text = Util.createEditable(targetModel.title)
+            area_edit_text.text = Util.createEditable("" + targetModel.radius)
+            targetModel.topic?.run {
+                selectedTopic(this)
+            }
+            text_select_topic.text = context.getString(R.string.topic)
+            text_area_length.text = context.getString(R.string.area_length)
```
Comment (amaury901130) [inline app/src/main/java/com/rootstrap/android/ui/activity/main/targetpoint/CreateTargetView.kt:245]: I think that you can do this in the next PR, but you're missing the topic icon, and the distance metric (m),
but like I said, just keep in mind for the next PR.

**Review by amaury901130 — APPROVED** [review]
Good work 💪

File: app/src/main/java/com/rootstrap/android/ui/activity/main/targetpoint/CreateTargetView.kt (hunk 1 above)
Comment (agustinkoll-rootstrap) [reply app/src/main/java/com/rootstrap/android/ui/activity/main/targetpoint/CreateTargetView.kt:245]: I added the topic icon in the topic selection

Comment (amaury901130) [reply app/src/main/java/com/rootstrap/android/ui/activity/main/targetpoint/CreateTargetView.kt:245]: nice 💪

**Pull request merged**